content.md

```diff
@@ -1,125 +1,126 @@
-
-### University of Virginia, School of Engineering & Applied Science
-#### B.S. with Honors in Computer Science - May 2005<br/> Also completed requirements for B.A. in Cognitive Science
-Thesis: **“NJTopol: Adding Topology to Parallelized Neural Simulation”**  
-Publications: Levy, William B., et al. **"The formation of neural codes in the hippocampus: trace conditioning as a prototypical paradigm for studying the random recoding hypothesis."** *Biological cybernetics* 92.6 (2005): 409-426.
+# Aprotim Sanyal 
+ 
+jobs@aprotim.com   
+(310) 243-6230 
+ 
+Employment 
+------------ 
+### Edgecast - **Principal Software Engineer, SRE** - *April 2020 - July 2022* 
+ 
+Rather than focusing on a single service, as a member of a small SRE task force for the entire organization, I identified areas of cross-cutting concern where we could act as a catalyst to amplify effort across other engineering teams and led several of these efforts. 
+ 
+- **Built organization-wide best practices.** My primary responsibility as an experienced SRE was evaluating existing procedures and tools 
+  across EdgeCast to create consistent processes and tooling in line with best practices. 
+  - **Evangelized SRE best practices to leadership.** Educated senior directors and VPs on the value of **SLIs/SLOs, error budgets, 
+    consistent tooling and practices**, etc. to build a better data-driven platform. 
+  - **Established SLIs and SLOs across teams.** Having established the value of SL\[IO\]s, I took 
+    on the large task of meeting with service teams across the org to understand their services and dependencies. I explained what makes useful, actionable SLIs, and coached team representatives through writing and instrumenting new SLIs and SLOs in line with these standards. 
+    This enabled the platform as a whole to move away from a reactive, incident-driven maintenance model towards proactive, error-budget-driven development. 
+  - **Created and documented escalation best practices for oncall engineers.** 
+    - **Establiblished standard oncall response SLAs and expectations.** After noticing differing and undocumented oncall response across teams different teams, I took on the task of creating common expectations across teams. I empaneled meetings with various stakeholders (NOC, service owners, leadership) to understand each team's implicit expectations for escalations (both their own and other teams'). Based on this input, I set up example service tiers with **specific SLAs around response time**, **definitions** for what "response" entailed, and **standards** for how to publish each team's oncall SLA and escalation procedures where they could be seen by others. 
+    - When onboarding the main CDN dev team onto the a shared rotation with SREs, I documented oncall expectations in line with the standards I created above. 
```

Comment:
Onboarded CDN dev team to shared SRE rotation, documenting expectations and process in line with newly established standards, providing an example process to the organization